src/arithmetic.jl

```diff
@@ -500,14 +502,14 @@ div!(v::Taylor1, b::NumberNotSeries, a::Taylor1, k::Int) =
 
 @inline function div!(c::TaylorN, a::TaylorN, b::TaylorN, k::Int)
     if k==0
-        @inbounds c[1] = a[1] / constant_term(b)
+        @inbounds c[0] = a[0] / constant_term(b)
```

Comment:
Is `constant_term(b)` not just `b[0]`?

Reply:
The difference is subtle; `b[0]` is a HomogeneousPolynomial of order 0, while `constant_term(b)` is just the (numeric) coefficient; using `b[0]` throws an error because dividing by `HomogeneousPolynomials`s is not defined.